src/components/roar-page/useHeader.tsx

```diff
@@ -1,8 +1,25 @@
+/*
+  Returns a header and associated objects/functions with the following capabilities:
+    1. Links that can scroll to various internal sections. The caller must then use the
```

Comment:
Can you please add the name of the variable before each description? I don't think the second point is relevant here

Reply:
Dramatically simplified the function and updated the comment